apps/src/sites/studio/pages/components/editorRedux.js

```diff
@@ -0,0 +1,127 @@
+/* global levelKeyList */
+
+import _ from 'lodash';
+
+function updatePositions(node) {
+  for (var i = 0; i < node.length; i++) {
+    node[i].position = i + 1;
+  }
+}
+
+let newStageId = -1;
+let newLevelId = -1;
+
+export default function reducer(state, action) {
```

Comment:
Looks like `_.cloneDeep(undefined)` returns `undefined`.  I'm surprised Redux needs an initial state when I'm explicitly passing in the initial state when I create the store.